docs/install.rst

```diff
@@ -148,17 +148,9 @@ this case you may consider using the ``--user`` option to install the package
 into your home directory. You can read more about how to do this in the `pip
 documentation <https://pip.pypa.io/en/stable/user_guide/#user-installs>`_.
 
-{% if is_development %}
-
-Alternatively, if you intend to do development on other software that uses
-``astropy``, such as an affiliated package, consider installing ``astropy`` into a :ref:`virtualenv<using-virtualenv>`.
-
-{%else%}
-
 Alternatively, if you intend to do development on other software that uses
-``astropy``, such as an affiliated package, consider installing ``astropy`` into a `virtualenv <https://docs.astropy.org/en/latest/development/workflow/virtualenv_detail.html>`__.
-
-{%endif%}
+``astropy``, such as an affiliated package, consider installing ``astropy``
+into a :ref:`virtualenv <astropy-dev:virtual_envs>`.
```

Comment:
This rendered correctly to https://docs.astropy.org/en/latest/development/workflow/virtual_pythons.html#virtual-envs